src/main/java/Ui.java

```diff
@@ -7,7 +7,7 @@
     // Strings that Duke will output
     private static final String PRINTED_LONG_LINE = "____________________________________________________________";
```

Comment:
Oh wow, this is an interesting way to print out your messages. Thanks for that! :)